test/integration/smoke/test_internal_lb.py

```diff
@@ -286,9 +302,12 @@ def setUpClass(cls):
                    %s" % (cls.account.name,
                           cls.account.id))
 
-        cls.cleanup = [cls.template, cls.account, cls.compute_offering]
+        cls._cleanup = [cls.template, cls.account, cls.compute_offering]
```

Comment:
Hi @borisroman

I added a setUp() and tearDown() methods that are called before/after each test. By doing so, we make sure that each test is done in isolation.

The tearDown() method uses the cleanup list, whilst the tearDownClass() method uses the _cleanup list.

The tearDownClass() is called only when all the tests are done.

:)